app/src/main/res/values/strings.xml

```diff
@@ -40,10 +39,7 @@
     <string name="app_bottomBar_about_label">About</string>
 
     <!-- Tokens -->
-    <string name="app_tokens_floatFormat_label" translatable="false">\u200e%sf</string>
-    <string name="app_tokens_dpFormat_label" translatable="false">\u200e%s dp</string>
```

Comment:
Why did we choose to remove`app_tokens_xxxFormat_label` resources?

Reply:
Because they will be identical for all languages and other platforms don't have these keys. It could be a problem when we will use POWO.